src/components/activity/ActivitySpansTable.svelte

```diff
@@ -0,0 +1,33 @@
+<svelte:options immutable={true} />
+
+<script lang="ts">
+  import type { ColDef, ColumnState } from 'ag-grid-community';
+  import type { Span, SpanId } from '../../types/simulation';
+  import BulkActionDataGrid from '../ui/DataGrid/BulkActionDataGrid.svelte';
+  import type DataGrid from '../ui/DataGrid/DataGrid.svelte';
+
+  export let columnDefs: ColDef[];
+  export let columnStates: ColumnState[] = [];
+  export let dataGrid: DataGrid = undefined;
+  export let selectedSpanId: SpanId | null = null;
+  export let spans: Span[] = [];
+
+  function getRowId(span: Span): SpanId {
+    return span.id;
+  }
+</script>
+
+<BulkActionDataGrid
```

Comment:
Does this need to be a BulkActionDataGrid if selecting the spans don't do anything and you can't bulk delete?